src/Recognizer.py

```diff
@@ -73,5 +74,15 @@ def __init__(self):
         super().__init__(supported_entity="POSTNOMINAL", patterns=patterns)
 
     def find(self, text, entities=None):
-        def find(self, text, entities=None):
-            return find_matches(self.patterns, text, self.supported_entity)
\ No newline at end of file
+            return find_matches(self.patterns, text, self.supported_entity)
+        
+class SSNRecognizer(PatternRecognizer):
+    def __init__(self):
+        patterns = [
+            Pattern("SSN",r"(\*\*\*\-\*\d\-\d{4})\b",score=1.0)
```

Comment:
```suggestion
            Pattern("SSN",r"((\*\*\*\-\*\d\-\d{4})|(\d{3}\-\d{2}\-\d{4}))\b",score=1.0)
```

I know the ssn is in a weird format in this file, but can you OR your regex pattern with a regex pattern to catch a normal SSN, I made a suggestion of what I mean, but I don't know if its syntactically correct